### ISSUE TITLE
Reading field names from .proto files

### ISSUE THREAD

**Issue by elahmo** [pawitp/protobuf-decoder#68]
Hey! Thanks for the great and fast little app.

I am wondering, how hard would it be to add support to upload a .proto file alongside the hex string? 

That way, the output could be enhanced with the field names and would be even more usable.

I would be happy to contribute to this if you could share some guidelines.

**Comment by pawitp** [comment]
Hi. Have you looked at other tools available on the Internet that can decode Protobuf using proto files already such as https://yura415.github.io/js-protobuf-encode-decode/ or https://www.protobufpal.com/ (note: I have no tested them).

I've built this tool because I couldn't find any available tool to decode Protobuf **without** proto files. While I don't mind accepting contributions if you would like to add support, if another tool is already fit for purpose, perhaps it would be easier to just use that.

**Comment by elahmo** [comment]
Yeah, I have checked some but they were quite crude and I didn't manage to get the result I was hoping for. For the first link you shared, it seems to require some manual changes to simple protos I have, but it does the trick. I shall use that then, thanks!
